### ISSUE TITLE
Adding Razor page doesn't use correct default namespace or ask me for name of page

### ISSUE THREAD

**Issue by timheuer** [microsoft/vscode-dotnettools#95]
Type: <b>Bug</b>

When adding a new item and choosing Razor page I wasn't asked what to name the file.  And the resulting file didn't have the right namespace for the page.

Extension version: 0.1.83
VS Code version: Code 1.79.0 (b380da4ef1ee00e224a15c1d4d9793e27c2b6302, 2023-06-07T14:26:35.552Z)
OS version: Windows_NT x64 10.0.22621
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|AMD Ryzen 9 3900X 12-Core Processor             (24 x 3793)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|undefined|
|Memory (System)|127.92GB (89.92GB free)|
|Process Argv|--crash-reporter-id d2c91f49-7cf5-4a88-bfb8-425825bf92ad|
|Screen Reader|no|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
vserr242:30382549
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
vscoreces:30445986
vscod805:30301674
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
cmake_vspar411:30581797
vsaa593cf:30376535
pythonvs932:30410667
cppdebug:30492333
vsclangdc:30486549
c4g48928:30535728
dsvsc012cf:30540253
pynewext54:30695312
azure-dev_surveyone:30548225
vscccc:30610679
2e4cg342:30602488
pyind779:30671433
89544117:30613380
pythonsymbol12:30671437
showlangstatbar:30737416
vsctsb:30748421
pythonms35:30701012
03d35959:30757346
57b77579:30736110
pythonfmttext:30731395
pythoncmvfstrcf:30756944
fixhidewlkth:30730051
hidesbindicator:30760978
pythongtdpathcf:30739705
dh2dc718:30763024

```

</details>

**Comment by smitpatel** [comment]
`dotnet new razor` doesn't take file name argument, it creates a default file with specific name always so we don't ask file name to user. I am not sure about namespace part of issue.

**Comment by arkalyanms** [comment]
Starting with Smit to reroute. (Makes it easier to get a better handle of leftover bugs to triage)

cc: @phil-allen-msft for razor impact.

**Comment by timheuer** [comment]
> `dotnet new razor` doesn't take file name argument, it creates a default file with specific name always so we don't ask file name to user. I am not sure about namespace part of issue.

It actually does: `dotnet new page -n MyPage -p:n MyNamespace.Pages`

**Comment by smitpatel** [comment]
Thanks @timheuer - Looks like each template has different set of arguments and I guess the one I tried didn't use -n flag. We will evaluate all available template options for each template we are supporting and add additional prompts for user to choose how the template would run.

**Comment by smitpatel** [comment]
Fixed in C# dev kit prerelease version 0.5.2
